rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/package.scala

```diff
@@ -22,5 +22,14 @@ package object sqs {
         case None    => reject(MissingQueryParamRejection(n))
       }
     }
+
+    def requiredUrlParam(n: String): Directive1[String] = {
+      p.get(n) match {
+        case Some(v) => provide(v)
+        case None    => reject(WrongURLFormatRejection(n))
```

Comment:
hm well if the parameter is absent, then the format isn't wrong, it's missing, no? :)

Reply:
Yes, changed back to `MissingQueryParamRejection` :)

Reply:
Yes, resulting exception is still the same.

Reply:
I mean, the method `requiredUrlParam` gives a `WrongURLFormatRejection` which is not very logical

Reply:
Modified solution, added new directive.